saleor/graphql/schema.graphql

```diff
@@ -1752,6 +1790,41 @@ enum DiscountValueTypeEnum {
   PERCENTAGE
 }
 
+enum DistanceUnitsEnum {
```

Comment:
@patrys Could you review these units? We need to define them for measurement attributes, but I think we don't need that many. Some look obsolete like "chain" or "fathom".

Reply:
See my comments for area units above. The same comments apply to their non-squared cousins here.

Reply:
Ok, so if so many units are useless we should go back to defining available units by ourselves than fetching them from the package.